gradient/api_sdk/clients/experiment_client.py

```diff
@@ -107,33 +126,53 @@ def create_multi_node(
             parameter_server_registry_container_user=None,
             parameter_server_registry_password=None,
     ):
-        """Create multi node experiment
-
-        :param str name:
-        :param str project_id:
-        :param str experiment_type_id:
-        :param str worker_container:
-        :param str worker_machine_type:
-        :param str worker_command:
-        :param int worker_count:
-        :param str parameter_server_container:
-        :param str parameter_server_machine_type:
-        :param str parameter_server_command:
-        :param int parameter_server_count:
-        :param str ports:
-        :param str workspace_url:
-        :param str working_directory:
-        :param str artifact_directory:
-        :param str cluster_id:
-        :param dict experiment_env:
-        :param str model_type:
-        :param str model_path:
-        :param str worker_container_user:
-        :param str worker_registry_username:
-        :param str worker_registry_password:
-        :param str parameter_server_container_user:
-        :param str parameter_server_registry_container_user:
-        :param str parameter_server_registry_password:
+        """
+        Create multi node experiment
+
+        *EXAMPLE*::
+
+            gradient experiments create singlenode
```

Comment:
this whole section needs to be a multinode example :)

Reply:
thanks :)